### PR TITLE
Remove dependency on jQuery for boot

### PR DESCRIPTION
**Pull request by cibernox**
@teddyzeenny  Copy-pasted the utility as this file is isolated.

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -6333,11 +6333,7 @@ spdx-license-ids@^1.0.2:
   version "1.2.2"
   resolved "https://registry.yarnpkg.com/spdx-license-ids/-/spdx-license-ids-1.2.2.tgz#c9df7a3424594ade6bd11900d596696dc06bac57"
 
-sprintf-js@^1.0.3:
-  version "1.1.1"
-  resolved "https://registry.yarnpkg.com/sprintf-js/-/sprintf-js-1.1.1.tgz#36be78320afe5801f6cea3ee78b6e5aab940ea0c"
-
-sprintf-js@~1.0.2:
+sprintf-js@^1.0.3, sprintf-js@~1.0.2:
```
Comment (teddyzeenny) [inline yarn.lock]: Are these changes intended?

Comment (cibernox) [inline yarn.lock]: Not particularly, but I guess that they don't harm.
